pkg/volume/util/storageclass.go

```diff
@@ -54,10 +54,17 @@ func GetDefaultClass(lister storagev1listers.StorageClassLister) (*storagev1.Sto
 	if len(defaultClasses) == 0 {
 		return nil, nil
 	}
+
+	sort.Slice(defaultClasses, func(i, j int) bool {
+		if defaultClasses[i].CreationTimestamp.UnixNano() == defaultClasses[j].CreationTimestamp.UnixNano() {
+			return defaultClasses[i].Name < defaultClasses[j].Name
+		}
+		return defaultClasses[i].CreationTimestamp.UnixNano() > defaultClasses[j].CreationTimestamp.UnixNano()
+	})
 	if len(defaultClasses) > 1 {
-		klog.V(4).Infof("GetDefaultClass %d defaults found", len(defaultClasses))
-		return nil, errors.NewInternalError(fmt.Errorf("%d default StorageClasses were found", len(defaultClasses)))
+		klog.V(4).Infof("%d default StorageClasses were found, choosing the newest: %s", len(defaultClasses), defaultClasses[0].Name)
```

Comment:
Also put in release note (and maybe the PR description) that the newest is preferred?

Should we document this in the API?

Reply:
Strictly speaking this is optional and not part of the API :)